### PR TITLE
Update Codeowners

### PR DESCRIPTION
**Pull request by rzwiefel**
CODEOWNERS only works for committers. 😭 

Removing non-committers.

### CODE REVIEW

**Comment by vinamartin** [comment]
😞 I was really hoping this would be a great contribution to our process. Oh well.

**Comment by rzwiefel** [comment]
build now

**Comment by stustison** [comment]
All your codeowner are belong to me!

**Comment by bdeining** [comment]
build now

File: CODEOWNERS [hunk 1]
```diff
@@ -55,28 +28,18 @@ catalog/spatial/csw/*           @bdeining
 
 catalog/spatial/registry/*      @clockard
 
-catalog/ui/catalog-ui-search/src/main/java/*		@rzwiefel
-
 catalog/ui/search-ui/standard/*		@stustison
 
+catalog/ui/catalog-ui-search/* @andrewkfiedler
```
Comment (clockard) [inline CODEOWNERS]: Should @andrewkfiedler  be on the same line with stustision

Comment (rzwiefel) [inline CODEOWNERS]: catalog-ui-search and search-ui/standard are separate modules.

**Pull request merged**